src/zope/viewlet/metaconfigure.py

```diff
@@ -113,9 +113,7 @@ def viewletDirective(
                 "Attribute and template cannot be used together.")
 
         # Note: The previous logic forbids this condition to evere occur.
```

Comment:
Typo: evere -> ever.  Actually, the whole clause should probably be rewritten to something like "forbids this condition from ever occurring", I think, but IANANS (I Am Not A Native English Speaker).

Reply:
Yup, typo (not mine!), and it could be phrased better.

Reply:
Sorry, didn't mean to throw any blame your way!  Just noticed an opportunity for improvement near one of the actual changes.